### PR TITLE
Redeem modal UI clean up

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

- Clean up redeem/burn modal UI
- Fix Burn modal when redemption rate is 0%

## Screenshots or screen recordings

| state | before | after |
| --- | --- | --- |
| redeemable state | Screen Shot 2022-08-01 at 2 12 57 PM | Screen Shot 2022-08-01 at 2 08 52 PM |
| no overflow | Screen Shot 2022-08-01 at 2 15 56 PM | Screen Shot 2022-08-01 at 11 40 50 AM |
| 0% redemption rate |Screen Shot 2022-08-01 at 2 16 53 PM | Screen Shot 2022-08-01 at 2 02 32 PM |

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by tomquirk** [comment]
@wraeth-eth could yu please review? 🙏

File: src/components/v2/V2Project/V2ManageTokensSection/V2RedeemModal.tsx [hunk 1]
```diff
@@ -192,80 +225,53 @@ export default function V2RedeemModal({
...
-              accessory={
-                <InputAccessoryButton
-                  content={t`MAX`}
-                  onClick={() => setRedeemAmount(fromWad(totalBalance))}
-                />
+          <Form form={form} layout="vertical">
+            <Form.Item
+              label={
+                canRedeem ? (
+                  <Trans>Tokens to redeem</Trans>
```
Comment (johnnyd-eth) [inline src/components/v2/V2Project/V2ManageTokensSection/V2RedeemModal.tsx:247]: **Praise**: great addition here

File: src/components/v2/V2Project/V2ManageTokensSection/V2RedeemModal.tsx [hunk 2]
```diff
@@ -192,80 +225,53 @@ export default function V2RedeemModal({
...
               </div>
             </Trans>
           </p>
           <p style={statsStyle}>
             <Trans>
-              Currently worth:{' '}
+              Redemption value:{' '}
               <span>
                 <CurrencySymbol currency="ETH" />
                 {formatWad(maxClaimable, { precision: 4 })}
```
Comment (johnnyd-eth) [inline src/components/v2/V2Project/V2ManageTokensSection/V2RedeemModal.tsx:236]: **Issue**: Think we need to make this the same precision as the redeem input `extra` (which is to 8dp at `RedeemModal:232`)

**Review by johnnyd-eth — CHANGES_REQUESTED** [review]
(precision mismatches)

File: src/components/v2/V2Project/V2ManageTokensSection/V2RedeemModal.tsx (hunk 2 above)
Comment (tomquirk) [reply src/components/v2/V2Project/V2ManageTokensSection/V2RedeemModal.tsx]: @johnnyd-eth yes v good pickup.

I did 2 things:
- improved ETHAmount some more. For low values, the tooltip on the `~0` ETH amount shows an 8-precision formatted amount.
- Used ETHAmount in the "you will receive" text.

| super low | normal |
| --- | --- |
| Screen Shot 2022-08-03 at 10 23 39 PM | Screen Shot 2022-08-03 at 10 23 46 PM |

**Review by johnnyd-eth — APPROVED** [review]
Sweet looks good! Merge once those conflicts are sorted :)

**Review by wraeth-eth — APPROVED** [review]

**Pull request merged**